compute/compute_test.go

```diff
@@ -8,158 +8,121 @@ package compute
 import (
 	"context"
 	"flag"
+	"fmt"
 	"log"
 	"os"
+	"strings"
 	"testing"
```

Comment:
>esting" [](start = 3, length = 7)

maybe move back to separate folders

Reply:
oops, forgot to do this. will do it in next update.

Reply:
tracked in #210